android/clientlib-ktx/src/main/java/com/solana/mobilewalletadapter/clientlib/LocalAdapterOperations.kt

```diff
@@ -24,6 +29,22 @@ class LocalAdapterOperations(
         }
     }
 
+    override suspend fun authorize(
+        identityUri: Uri,
+        iconUri: Uri,
+        identityName: String,
+        chain: String,
+        authToken: String?,
```

Comment:
Same as previous comment, does it make sense to provide default nulls here?

Reply:
Gotta be in the interface